scipy/interpolate/ndgriddata.py

```diff
@@ -132,6 +132,13 @@ def griddata(points, values, xi, method='linear', fill_value=np.nan,
         incommensurable units and differ by many orders of magnitude.
 
         .. versionadded:: 0.14.0
+        
+    Returns
+    -------
+    
+    ip: ndarray
+        Interpolation points of dimensions xi and based on the method
```

Comment:
How about something like "Array with the same shape as `xi` containing the interpolated values."?

Reply:
Wait, ignore that comment.  The returned array is not "the same shape as `xi`".

Reply:
It turns out the description of `xi` in the "Parameters" section is misleading.  It says `"xi : 2-D ndarray of float or tuple of 1-D array, shape (M, D)"`, but if `xi` is a tuple, the values in the tuple do *not* have to be 1-D.  Just look at the example, where `grid_x` and `grid_y` are 2-D.  In fact, the arrays in the tuple can have arbitrary shape, as long as the shapes are compatible for broadcasting.  And if `xi` is a numpy array, it does not have to be 2-D.  Apparently just the *last* dimension of `xi` must be `D`.